### PR TITLE
Moving SP docs to public repo

### PR DESCRIPTION
**Pull request by cs-namrata-mathur**

### CODE REVIEW

File: docs/res/pbDesigner_Advisorbot.png
Comment (cs-yash-patodkar) [inline docs/res/pbDesigner_Advisorbot.png:1]: Change this image has an older version logo

File: docs/res/AdvisorFlyout.png
Comment (cs-yash-patodkar) [inline docs/res/AdvisorFlyout.png:1]: Change the image with latest logo

File: docs/res/pbDesigner_Advisorbot.png
Comment (cs-namrata-mathur) [reply docs/res/pbDesigner_Advisorbot.png:1]: Removed the image it was not being used in the doc

File: docs/res/AdvisorFlyout.png
Comment (cs-namrata-mathur) [reply docs/res/AdvisorFlyout.png:1]: Removed the image it was not being used in the doc

**Review by cs-abhinav-goel — APPROVED** [review]
Looks good

**Pull request merged**